com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceShadows.cs

```diff
@@ -49,11 +49,17 @@ public override void AddRenderPasses(ScriptableRenderer renderer, ref RenderingD
                 return;
             }
 
+            bool isDeferredRenderingMode = renderer is UniversalRenderer && ((UniversalRenderer)renderer).renderingMode == RenderingMode.Deferred;
+
```

Comment:
Shouldn't we move this check so it's inside the` if (shouldEnqueue)` as that's the only place it's used?

Reply:
Ok, let me do it